rgw/v2/tests/s3_swift/test_bucket_policy_with_tenant_user.py

```diff
@@ -253,6 +274,38 @@ def test_exec(config, ssh_con):
             else:
                 raise TestExecError("bucket policy creation failed")
 
+            if config.test_ops.get("put_bucket_website", False):
+                reusable.put_bucket_website(rgw_tenant1_user1_c, bucket.name)
+
+                log.info(
+                    "with non bucket owner of same tenant, perform put bucket wesite"
+                )
+                reusable.put_bucket_website(rgw_tenant1_user2_c, bucket.name)
+                reusable.put_bucket_website(rgw_tenant1_user3_c, bucket.name)
+
+                log.info(
+                    "with non bucket owner of different tenant, perform put bucket wesite"
+                )
```

Comment:
Same as above, can we improve the log info msg

Reply:
@mkasturi18 , made changes